frappe/commands/site.py

```diff
@@ -293,16 +304,20 @@ def use(site, sites_path='.'):
 
 @click.command('backup')
 @click.option('--with-files', default=False, is_flag=True, help="Take backup with files")
+@click.option('--encrypt', default=False, is_flag=True, help="Encrypt dumped sql file")
 @pass_context
 def backup(context, with_files=False, backup_path_db=None, backup_path_files=None,
-	backup_path_private_files=None, quiet=False):
+	backup_path_private_files=None, quiet=False, encrypt=False):
 	"Backup"
 	from frappe.utils.backups import scheduled_backup
 	verbose = context.verbose
+	automated_backup = False
+	if len(context.sites) != 1:
+		automated_backup = True
```

Comment:
Just to be clear, this is the difference between `bench --site site-name backup` and  `bench backup` right?

In that doesn't it feel wrong to implicitly assume every case of `bench backup` is triggered with a cronjob?

Reply:
That line is not supposedly assuming that the bench backup is triggered with a cronjob but to identify if it's using bench backup through a cronjob. The line for backup in cronjob requires all sites to be dumped, so if it's using the cronjob for backup.. the process will know that it's an automated process since it got multiple sites when ```context.sites``` got more than one site. then the process will identify if the dumped SQL file will be encrypted or not.